package/CHANGELOG

```diff
@@ -16,16 +16,17 @@ The rules for this file:
 ??/??/?? tylerjereddy, richardjgowers, IAlibay, hmacdope, orbeckst, cbouy,
          lilyminium, daveminh, jbarnoud, yuxuanzhuang, VOD555, ianmkenney,
          calcraven，xiki-tempula, mieczyslaw, manuel.nuno.melo, PicoCentauri,
-         hanatok, rmeli, aditya-kamath, tirkarthi
+         hanatok, rmeli, aditya-kamath, tirkarthi, Henry Kobin
```

Comment:
```suggestion
         hanatok, rmeli, aditya-kamath, tirkarthi, HenryKobin
```

The CHANGELOG asks for your GitHub username. Instead, please add your name to the AUTHORS files.